src/components/GameSnack/index.tsx

```diff
@@ -48,7 +48,7 @@ const GameSnack = () => {
 
     return (
         <Snackbar
-            anchorOrigin={{ vertical: "bottom", horizontal: "center" }}
+            anchorOrigin={{ horizontal: "center", vertical: "bottom" }}
```

Comment:
Let's multi-line this object.

```suggestion
            anchorOrigin={{
                horizontal: "center",
                vertical: "bottom",
            }}
```